pandas/_libs/tslibs/strptime.pyx

```diff
@@ -723,6 +775,148 @@ cdef tzinfo _parse_with_format(
     return tz
 
 
+def _array_strptime_object_fallback(
+    ndarray[object] values,
```

Comment:
Might need to be careful with performance of this during transition from Cython 0.29 to Cython 3. Definitely if this was just `ndarray values` you would lose performance if you didn't use `PySequence_GetItem` in the loop; I'm not sure if declaring it as `ndarray[object] values` makes a difference

Reply:
should i update this to use PySequence_GetItem?  im not too concerned with this function since it will be removed in 3.0 anyway

Reply:
If it's going away soon probably not worth the effort